### DISCUSSION TITLE
Can ONNX get gradient directly through API like torch.autograd.grad(...)??

### DISCUSSION THREAD

**Discussion by FdyCN** [microsoft/onnxruntime#11375] [Other Q&A]
I'm new in ONNX，i found a gradient-related demo in :
https://github.com/onnx/onnx/blob/35092895d9bf3592e58f4710d098f8131afef259/onnx/backend/test/case/model/gradient.py#L11

However, it isn't seemed like what i want. I wonder if there is any python API which can get gradient directly. like torch.autograd.grad()？？？
**System information**
- ONNX Runtime version (you are using): 1.9.0

**Comment by Fansomone** [comment]
The same question, is there a C++ API in onnxruntime like torch::autograd::grad()?

**Comment by FelixBiermann** [comment]
Any development?
